components/openzwave-adapter/src/lib.rs

```diff
@@ -182,8 +182,8 @@ pub struct OpenzwaveAdapter {
     version: [u32; 4],
     ozw: ZWaveManager,
     node_map: IdMap<ServiceId, Node>,
-    getter_map: IdMap<Getter, ValueID>,
-    setter_map: IdMap<Setter, ValueID>,
+    getter_map: IdMap<Channel, ValueID>,
+    setter_map: IdMap<Channel, ValueID>,
```

Comment:
I think we'll want to merge these maps into the same map; probably more than you're willing to do in this patch but please file a bug.